addons/web/controllers/binary.py

```diff
@@ -236,7 +244,7 @@ def upload_attachment(self, model, id, ufile, callback=None):
         '/web/binary/company_logo',
         '/logo',
         '/logo.png',
-    ], type='http', auth="none", cors="*")
+    ], type='http', auth='none', cors='*')
```

Comment:
I think this one too can be made readonly
```suggestion
    ], type='http', auth='none', cors='*', readonly=True)
```